repos.yaml

```diff
@@ -0,0 +1,157 @@
+repos:
+  - name: Alpine
+    help: |
```

Comment:
Is the "help" field optional?

Reply:
yes

Reply:
After internal discussion, yes its optional